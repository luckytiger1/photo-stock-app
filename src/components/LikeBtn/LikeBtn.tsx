import React from 'react';

const LikeBtn = ({ itemClass, addToFavorites, icon }: any) => {
  return (
    <button className={itemClass} type="button" onClick={addToFavorites}>
      <img src={icon} alt="like" />
    </button>
  );
};

export default LikeBtn;
