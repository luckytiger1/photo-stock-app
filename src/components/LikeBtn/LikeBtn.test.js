import React from 'react';
import { shallow } from 'enzyme';
import { createSerializer } from 'enzyme-to-json';
import LikeBtn from './LikeBtn';

it('<LikeBtn /> renders correctly', () => {
  expect.addSnapshotSerializer(createSerializer({ mode: 'deep' }));
  expect(shallow(<LikeBtn />)).toMatchSnapshot();
});
