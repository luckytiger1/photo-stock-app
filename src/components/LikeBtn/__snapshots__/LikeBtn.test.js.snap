// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LikeBtn /> renders correctly 1`] = `
<button
  type="button"
>
  <img
    alt="like"
  />
</button>
`;
